specification/playwrighttesting/PlaywrightTesting.Management/examples/Quotas_Get.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Quotas_Get",
+  "operationId": "Quotas_Get",
+  "parameters": {
+    "api-version": "2023-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "location": "eastus",
+    "name": "ScalableExecution"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.AzurePlaywrightService/locations/eastus/quotas/ScalableExecution",
+        "name": "ScalableExecution",
+        "type": "Microsoft.AzurePlaywrightService/Locations/Quotas",
```

Comment:
resource type names should be capitalized with camelcase
"Microsoft.AzurePlaywrightService/Locations/Quotas",